contribute/code.rst

```diff
@@ -0,0 +1,158 @@
+Code
+====
+
+We're serious about code quality and stability. We have to be, Ubuntu Touch is used by many users on many different devices. Our quality requirements be intimidating for new contributors as we often ask for many rounds of changes during reviews. We understand that you've chosen to spend your time developing changes to Ubuntu Touch as a volunteer, and we thank you for that. We will respect you and your time during the review process. However, we ask that you respect our time as much as we respect yours. If you respect our time by following these guidelines, you should see a speedier, lower-friction review process.
```

Comment:
"be intimidating" -> "might be intimidating", I guess

Reply:
Arr, you don't like our review style matey?

Reply:
should that be a full stop after "We have to be." or maybe even an exclamation mark